package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "rollup-plugin-css-modules",
+  "version": "0.0.0",
+  "description": "Rollup support for standard CSS modules",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "build": "wireit",
+    "test": "wireit",
+    "format": "wireit"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
missing author, and probably nice to add some keywords too. can totally be done separately from this PR before publishing though.